From: Chris Hofstaedtler <zeha@debian.org>
Date: Wed, 13 Apr 2022 11:11:50 +0000
Subject: tests: fdisk/bsd: update expected output for ppc64le

Update expected test data after upstream MRs #1385 and #1386.
Upstream issue: https://github.com/util-linux/util-linux/issues/1655
---
 tests/expected/fdisk/bsd_0_64.LE | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/tests/expected/fdisk/bsd_0_64.LE b/tests/expected/fdisk/bsd_0_64.LE
index 3dbe623..9663f16 100644
--- a/tests/expected/fdisk/bsd_0_64.LE
+++ b/tests/expected/fdisk/bsd_0_64.LE
@@ -47,16 +47,16 @@ I/O size (minimum/optimal): 512 bytes / <removed> bytes
 Disklabel type: dos
 Disk identifier: <removed>
 
-Device     Boot Start   End Sectors Size Id Type
-<removed>1       2048  4095    2048   1M 83 Linux
+Device     Boot Start   End Sectors Id Type  Start-C/H/S End-C/H/S Attrs
+<removed>1       2048  4095    2048 83 Linux     0/32/33    0/65/1 
 -------------------
 
 Create 2nd primary partition
 00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 *
 000001b0  00 00 00 00 00 00 00 00  01 00 00 00 00 00 00 20  |............... |
-000001c0  21 00 83 41 01 00 00 08  00 00 00 08 00 00 00 04  |!..A............|
-000001d0  01 3e 83 13 41 36 00 10  00 00 00 40 00 00 00 00  |.>..A6.....@....|
+000001c0  21 00 83 41 01 00 00 08  00 00 00 08 00 00 00 41  |!..A...........A|
+000001d0  02 00 83 46 05 01 00 10  00 00 00 40 00 00 00 00  |...F.......@....|
 000001e0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 000001f0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 55 aa  |..............U.|
 00000200  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
@@ -66,8 +66,8 @@ Set 2nd partition type
 00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 *
 000001b0  00 00 00 00 00 00 00 00  01 00 00 00 00 00 00 20  |............... |
-000001c0  21 00 83 41 01 00 00 08  00 00 00 08 00 00 00 04  |!..A............|
-000001d0  01 3e a5 13 41 36 00 10  00 00 00 40 00 00 00 00  |.>..A6.....@....|
+000001c0  21 00 83 41 01 00 00 08  00 00 00 08 00 00 00 41  |!..A...........A|
+000001d0  02 00 a5 46 05 01 00 10  00 00 00 40 00 00 00 00  |...F.......@....|
 000001e0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 000001f0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 55 aa  |..............U.|
 00000200  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
@@ -82,17 +82,17 @@ I/O size (minimum/optimal): 512 bytes / <removed> bytes
 Disklabel type: dos
 Disk identifier: <removed>
 
-Device     Boot Start   End Sectors Size Id Type
-<removed>1       2048  4095    2048   1M 83 Linux
-<removed>2       4096 20479   16384   8M a5 FreeBSD
+Device     Boot Start   End Sectors Id Type    Start-C/H/S End-C/H/S Attrs
+<removed>1       2048  4095    2048 83 Linux       0/32/33    0/65/1 
+<removed>2       4096 20479   16384 a5 FreeBSD      0/65/2    1/70/5 
 -------------------
 
 Create default BSD
 00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 *
 000001b0  00 00 00 00 00 00 00 00  01 00 00 00 00 00 00 20  |............... |
-000001c0  21 00 83 41 01 00 00 08  00 00 00 08 00 00 00 04  |!..A............|
-000001d0  01 3e a5 13 41 36 00 10  00 00 00 40 00 00 00 00  |.>..A6.....@....|
+000001c0  21 00 83 41 01 00 00 08  00 00 00 08 00 00 00 41  |!..A...........A|
+000001d0  02 00 a5 46 05 01 00 10  00 00 00 40 00 00 00 00  |...F.......@....|
 000001e0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 000001f0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 55 aa  |..............U.|
 00000200  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
@@ -144,8 +144,8 @@ Command (m for help):
 00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 *
 000001b0  00 00 00 00 00 00 00 00  01 00 00 00 00 00 00 20  |............... |
-000001c0  21 00 83 41 01 00 00 08  00 00 00 08 00 00 00 04  |!..A............|
-000001d0  01 3e a5 13 41 36 00 10  00 00 00 40 00 00 00 00  |.>..A6.....@....|
+000001c0  21 00 83 41 01 00 00 08  00 00 00 08 00 00 00 41  |!..A...........A|
+000001d0  02 00 a5 46 05 01 00 10  00 00 00 40 00 00 00 00  |...F.......@....|
 000001e0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
 000001f0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 55 aa  |..............U.|
 00000200  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
